client/extensions/woocommerce/app/stats/index.js

```diff
@@ -2,19 +2,40 @@
  * External dependencies
  */
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import Main from 'components/main';
 import StatsNavigation from './stats-navigation';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import StatsChart from './stats-chart';
 
-export default class Stats extends Component {
+class Stats extends Component {
```

Comment:
Not critical but we may want to rename this at some point. `StatsStore` perhaps (similar to `StatsSite` [here](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/stats/site.jsx#L30))?